api/src/schema.graphql

```diff
@@ -1,74 +1,207 @@
-# import Product from "./generated/prisma.graphql"
-# import ID from "./generated/prisma.graphql"
+## NOTE: This schema was generated by Graphback and should not be changed manually
 
-type Query {
-  products(
-    where: ProductWhereInput
-    orderBy: ProductOrderByInput
-    skip: Int
-    after: String
-    before: String
-    first: Int
-    last: Int
-  ): [Product]!
-  order(id: ID!): Order
+"""@model"""
+type Brand {
+  id: ID!
+
+  """@db.type: 'timestamp'"""
+  createdAt: DateTime!
+
+  """@db.type: 'timestamp'"""
+  updatedAt: DateTime!
+
+  """@db.unique"""
+  name: String!
+  slug: String!
+  url: String!
+
+  """@oneToMany field: 'brand', key: 'brandId'"""
+  products: [Product]
 }
 
-type UpdateOrderRowResponse {
-  order: Order!
-  row: OrderRow
+input BrandInput {
+  id: ID
+  createdAt: DateTime
+  updatedAt: DateTime
+  name: String
+  slug: String
+  url: String
+}
+
+scalar DateTime
+
+"""@model"""
+type DiscountCode {
+  id: ID!
+
+  """@db.type: 'timestamp'"""
+  createdAt: DateTime!
+
+  """@db.type: 'timestamp'"""
+  updatedAt: DateTime!
+  code: String!
+  name: String!
+  description: String!
+  type: DiscountCodeType!
+  amount: Float!
+
+  """@manyToOne field: 'discountCodes', key: 'orderId'"""
+  order: Order
+}
+
+input DiscountCodeInput {
+  id: ID
+  createdAt: DateTime
+  updatedAt: DateTime
+  code: String
+  name: String
+  description: String
+  type: DiscountCodeType
+  amount: Float
+  orderId: ID
+}
+
+enum DiscountCodeType {
+  Percentage
 }
 
 type Mutation {
-  createOrder: Order!
-  addProductToOrder(
-    orderId: String!
-    productId: String!
-    quantity: Int
-  ): OrderRow!
-  updateOrderRow(id: ID!, quantity: Int): UpdateOrderRowResponse
-  addDiscountCodeToOrder(orderId: ID!, code: String!): Order!
```

Comment:
addXtoX seems to be common pattern here? In our case it will be create discountCode (but it will return DiscountCode instead of Order.